MOBILE_API.xml

```diff
@@ -2213,7 +2219,9 @@
         <param name="numCustomPresetsAvailable" type="Integer" minvalue="1" maxvalue="100" mandatory="false" since="3.0">
             <description>The number of on-screen custom presets available (if any); otherwise omitted.</description>
         </param>
-        
+        <param name="menuLayoutsAvailable" type="MenuLayout" array="true" minsize="1" maxsize="1000" mandatory="false" since="6.0">
```

Comment:
It will be easiest to make this change once https://github.com/smartdevicelink/rpc_spec/pull/185 is merged as WindowCapabilities does not exist yet in the base branch of this PR.